pkg/sensors/tracing/generictracepoint.go

```diff
@@ -547,11 +552,18 @@ func (t *observerTracepointSensor) SpecHandler(raw interface{}) (*sensors.Sensor
 		spec = &s
 	}
 
+	var name string
+	if named, ok := raw.(interface{ Name() string }); ok {
+		name = named.Name()
```

Comment:
Can this happen? Shouldn't we have `Name()` in https://github.com/cilium/tetragon/blob/c9a0eaeefb40beef5d78f6872404c6e7309e756e/pkg/sensors/sensors.go#L140-L143?

Reply:
Hmm... I don't think so. Seems like a leftover. I'll remove it.

Reply:
removed